docs/conf.py

```diff
@@ -81,6 +81,18 @@
 # Activate autosectionlabel plugin
 autosectionlabel_prefix_document = True
 
+# sphinx-notfound-page
+# https://github.com/rtfd/sphinx-notfound-page
+notfound_context = {
+    'body': '''
+<h1>Page not found</h1>
+
+<p>Sorry, we couldn't find that page.</p>
+
+<p>Try using the search box or go to the homepage.</p>
+''',
+}
+
```

Comment:
This is just for the future, right? We need to add the extension if we want it to do anything.

Reply:
This is _just of our docs_ to start testing it.

The extension is installed in our requirements.

I can remove it if you consider that it's best to start testing this in another dumb project, though.